cloud/google/start_log.sh

```diff
@@ -37,6 +39,7 @@ for i in `seq ${LOG_NUM_REPLICAS}`; do
       --image container-vm \
       --disk name=${LOG_DISKS[${i}]},mode=rw,boot=no,auto-delete=yes \
       --tags log-node \
+      --scopes "monitoring,storage-ro,compute-ro,logging-write" \
```

Comment:
Just gets the instance permission to call the listed Cloud APIs.
Needed to add `monitoring`, but then I need to explicitly list all the others too.
I suppose we may not need `monitoring` if you chose `prometheus`, but it won't hurt.